core/doc_data.h

```diff
@@ -69,6 +69,18 @@ class DocData {
 		Vector<ArgumentDoc> arguments;
 		Vector<int> errors_returned;
 		bool operator<(const MethodDoc &p_method) const {
+			if (name.left(8) == "operator") {
```

Comment:
The fix looks good, but it makes the comment below obsolete, so it would be good to refactor a bit and make sure the comments are accurate and sufficient.

This could go as a special case under the `if (name == p_method.name)` too:
```cpp
if (name == p_method.name) {
	// Explanations
	if (name.begins_with("operator ")) {
		...
	} else { // Constructors
	
	}
}
return name < p_method.name;
```

Reply:
Done. Let me know if you think it needs any more comments.